### PR TITLE
Change version to v0.1

### PR DESCRIPTION
**Pull request by damithc**
Let's use the new version

### CODE REVIEW

File: src/main/java/Contacts0.java [hunk 1]
```diff
@@ -3,14 +3,16 @@
 
 public class Contacts0 {
 
+    public static final int MAX_CONTACTS_ALLOWED = 100;
```
Comment (damithc) [inline src/main/java/Contacts0.java]: Is this really better? :thinking:

**Comment by ananda-lye** [comment]
Update this

**Comment by DeetoMok** [comment]
Refactor codes

**Comment by ananda-lye** [comment]
Changed version number

**Pull request closed without merging**